src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -186,6 +206,10 @@ public String getShard() {
         return getConfig().getString(SHARD);
     }
 
+    public String getGTID() {
```

Comment:
```suggestion
    public String getGtid() {
```

Unless it's a static member, we use that style in general for abbreviations in member names (it's not done consistently 100% unfortunately).